qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -186,16 +186,10 @@ def __init__(self, *, win_id, private, parent=None):
         self._hbox.addWidget(self.keystring)
```

Comment:
`keystring` and `progress` (below) should probably be configurable as well?

Reply:
Done.